src/components/DownloadLink/DownloadLink.tsx

```diff
@@ -0,0 +1,29 @@
+import { Flex, FlexItem, FlexProps } from '@patternfly/react-core';
+import { DownloadIcon } from '@patternfly/react-icons';
+
+interface IDownloadLinkProps {
+  url: string;
+  title: string;
+  showIcon?: boolean;
+}
+
+const spaceItemsSm: FlexProps['spaceItems'] = { default: 'spaceItemsSm' };
+
+export const DownloadLink = ({ url, title, showIcon = true }: IDownloadLinkProps) => {
+  return (
+    <Flex spaceItems={spaceItemsSm}>
```

Comment:
The download link and icon were divided into two rows when the screen is narrow, consider combining it.
![image](https://github.com/project-ncl/pnc-web-ui-react/assets/6870455/d3a1e15e-5182-467f-9cef-01bcb473f92c)

Reply:
When the box is very small like on the screenshot above, I would propose applying different layout, that is already applied on other pages when layout is narrow, for example:

![image](https://github.com/project-ncl/pnc-web-ui-react/assets/17566891/320255a1-28f0-4237-9357-0eb060b70db5)

Reply:
@matedo1 it does actually work that way, but the screen needs to be more narrow for that layout to be applied.

Reply:
Hi @patrikk0123 , could you try to investigate a little bit to align the behavior(wide, medium, narrow) so that the situation on the screen would not appear? Thanks!